lib/basedriver/desired-caps.js

```diff
@@ -74,6 +74,12 @@ validator.validators.isNumber = function (value) {
     return null;
   }
 
+  // allow a string value
+  if (typeof value === 'string' && !Number.isNaN(value)) {
```

Comment:
Isn't this the same as just checking if `value` is a string? If `value` is a string, then `Number.isNaN(value)` will always be false.

Reply:
yep, my bad. Fixed